### DISCUSSION TITLE
How to submit a PR?

### DISCUSSION THREAD

**Discussion by fullphat** [samemory/homebridge-eufy-security-control#317] [Q&A]
Not a big GitHub user so could do with some guidance on how to submit a PR.  I have modified two files to fix live streaming via RTSP for the T8420, and also enabled the floodlights on the same camera.

When I try to open a PR, the bot auto-closes it saying I need to submit on the dev branch, but I can't find a dev branch.

Any help appreciated!

**Comment by schliemann** [comment]
@lenoxys knows most about the workflow. Maybe he can help.

**Comment by schliemann** [comment]
@fullphat I created a dev branch. Can you try and create a PR to it? I think something is wrong with the workflow we have. But need input from @lenoxys .

**Reply by fullphat** [reply]
No problem - I can see the dev branch now but I can't create a PR as it says it's identical to the master.  I'll see if I can push my mods to this branch and then hopefully I can do the PR.  Leave it with me. 👍🏼

**Reply by fullphat** [reply]
Ok, I've cloned the dev branch and have added my changes to the relevant files.  I can't push my changes though as I don't have write access so I'm a bit stuck again now.
